src/main/topLevelPages/myDashboard/newVirtualContributorWizard/useNewVirtualContributorWizard.tsx

```diff
@@ -574,6 +586,28 @@ const useNewVirtualContributorWizard = (): useNewVirtualContributorWizardProvide
     }
   };
 
+  const handleCreateExternal = async (externalVcValues: ExternalVcFormValues) => {
+    if (virtualContributorInput && myAccountId) {
+      virtualContributorInput.engine = externalVcValues.engine;
+
+      virtualContributorInput.externalConfig = {
+        apiKey: externalVcValues.apiKey,
+      };
+      if (externalVcValues.assistantId) {
+        virtualContributorInput.externalConfig.assistantId = externalVcValues.assistantId;
+      }
+      const createdVc = await executeMutation({
+        values: virtualContributorInput,
+        accountId: myAccountId,
+      });
+
+      // navigate to VC page
+      if (createdVc) {
+        navigate(`vc/${createdVc.nameID}`);
```

Comment:
let's use just `createdVc.profile.url` instead of nameID if possible

Reply:
Maybe I have a failed push - locally it's updated for me.